src/refiners/foundationals/clip/image_encoder.py

```diff
@@ -87,7 +76,7 @@ def __init__(
         self.patch_size = patch_size
         super().__init__(
             fl.Concatenate(
-                ClassEncoder(embedding_dim=embedding_dim, device=device, dtype=dtype),
+                ClassEncoder(embedding_dim, device=device, dtype=dtype),
```

Comment:
This should be `ClassEncoder(1, embedding_dim, device=device, dtype=dtype)` (missing dimension for this extra token)